app/routes.py

```diff
@@ -1,2 +1,182 @@
-from flask import Blueprint
+"""
+Something went very, very wrong and I had to scrap my original repository, 
+start over and copy paste my code in. Hence, the lack of commits. 
+I was beginning the deployment stage and accidently deleted the origin remote. 
+This lead me down a very dark path, trying different git commands that I didnt fully understand. 
+It was chaos.
 
+Anyways,
+I didn't have time to do docstrings.
+Also, I experimented with doubling up the route decorators since the functions were 
+pretty much the same for Task & Goals.
+Not sure if this is a no no in real life, or if there is a better way to do it. Let me know!
+"""
+
+
+from flask import Blueprint, jsonify, request, abort, g
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import requests
+import os
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+def slack_bot(text):
```

Comment:
Nice work creating a helper function. You might consider refactoring as an instance method on `task` given that the text has information about a specific task.